<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mp3-Tagger</string>
  </property>
  <widget class="QLabel" name="folderAlbum_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Папка с альбомом:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="folderAlbum_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="folderAlbum_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="coverAlbum_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="coverAlbum_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QLabel" name="coverAlbum_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Обложка альбома:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="artistAlbum_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="artist_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Исполнитель альбома:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameAlbum_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="albumname_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование альбома:</string>
   </property>
  </widget>
  <widget class="QLabel" name="genre_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Жанр альбома:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearAlbum_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Год выпуска альбома:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearAlbum_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m3u_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m3u_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m3u_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>m3u файл:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="result_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Поехали...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="genre_combobox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
